hasura/metadata/databases/default/tables/public_cosouls.yaml

```diff
@@ -17,6 +17,4 @@ select_permissions:
         - synced_at
         - token_id
         - updated_at
-      filter:
-        profile_id:
-          _eq: X-Hasura-User-Id
+      filter: {}
```

Comment:
@wingfeatherwave this change would make this table visible to everyone logged in, i think thats ok but a bit of a change.

Reply:
this is good